core/system/src/mpu/vmpu_armv7m.c

```diff
@@ -640,13 +640,14 @@ void vmpu_arch_init(void)
 
     /* show basic mpu settings */
     DPRINTF("MPU.REGIONS=%i\r\n", (uint8_t)(MPU->TYPE >> MPU_TYPE_DREGION_Pos));
-    DPRINTF("MPU.ALIGNMENT=0x%08X\r\n", g_vmpu_aligment_mask);
-    DPRINTF("MPU.ALIGNMENT_BITS=%i\r\n", vmpu_bits(g_vmpu_aligment_mask));
+    DPRINTF("MPU.ALIGNMENT=0x%08X\r\n", aligment_mask);
+    DPRINTF("MPU.ALIGNMENT_BITS=%i\r\n", vmpu_bits(aligment_mask));
 
     /* Perform sanity checks. */
-    assert(ARMv7M_MPU_REGIONS == ((MPU->TYPE >> MPU_TYPE_DREGION_Pos) & 0xFF));
-    assert(ARMv7M_MPU_ALIGNMENT_BITS == vmpu_bits(g_vmpu_aligment_mask));
-
+    if(ARMv7M_MPU_REGIONS != ((MPU->TYPE >> MPU_TYPE_DREGION_Pos) & 0xFF))
```

Comment:
Could you add a space after the `if` and the [1TBS braces](https://en.wikipedia.org/wiki/Indent_style#Variant:_1TBS)?

```C
if (condition) {
    code;
}
```

(also applies to line 649)